### DISCUSSION TITLE
Meeting

### DISCUSSION THREAD

**Discussion by jcgs2503** [stevengogogo/DSA_EmailSearcher#18] [General]
@stevengogogo I noticed that you opened several discussion threads, sorry for not replying immediately, I just recovered from a fever. Jeffery and I are sort of still out of the loop. Shall we set up a meeting via Google meet or Zoom to discuss so that we can work on the project more efficiently? Thanks.

@Jeffrey911106

**Comment by stevengogogo** [comment]
Sure! I am free on day time. Let's use Google Meet
Take care, stay healthy!

**Reply by jcgs2503** [reply]
Thanks, np. I'll ask @Jeffrey911106 when he is available.

**Reply by Jeffrey1106** [reply]
How about Saturday afternoon?

**Reply by jcgs2503** [reply]
I'm fine with that!

**Reply by stevengogogo** [reply]
See you then. @jcgs2503  could you schedule a meeting?

**Reply by jcgs2503** [reply]
Np.

**Comment by stevengogogo** [comment]
Before the meeting starts, we could read the problems first.
I have organized the files on NTU COOL to save your time:

#13  `(Downloaded on May 20th)`

**Reply by stevengogogo** [reply]
Merged PDF: [problem.pdf](https://github.com/stevengogogo/DSA_EmailSearcher/blob/2b414baf3af4f013686e4f057d9dc338608ec816/problem.pdf)

**Reply by stevengogogo** [reply]
My notes
- Problem Statement
    - [ ] Predefined data type: #24 
    - [ ] Helper function: #27 
    - [ ] Token and expression: #31 
- [ ] Teamwork #33

**Comment by stevengogogo** [comment]
Meeting note
-------------

https://hackmd.io/cY_I8x36QBOLisedOdzr_w
